code/game/gamemodes/clock_cult/clock_structures/ark_of_the_clockwork_justicar.dm

```diff
@@ -113,7 +113,7 @@
 /obj/structure/destructible/clockwork/massive/celestial_gateway/proc/mass_recall()
 	for(var/V in SSticker.mode.servants_of_ratvar)
 		var/datum/mind/M = V
-		if(!M.current.stat)
+		if(!M.current.stat != DEAD)
```

Comment:
```M.current.stat != DEAD```?

Reply:
Um, yes. That is generally the code to check if someone it not dead. This may come as a surprise.